README.md

```diff
@@ -1,31 +1,128 @@
+
+## Updating submodules
+
+Make sure to run `git submodule update --recursive` after you pull from the repo to update the submodules to the commits tracked in the repo.
+
+You can run e.g. `cd runtimes && git checkout <commit_hash> && cd - && git add runtimes && git commit` to update the commit that the runtimes submodule points to.
+
+## AHM Flows (manual steps)
+
+The cmd `just ahm <runtime>` use the `orchestrator` as main control flow to coordinate the usage of a mix of tools (e.g:[zombie-bite](), [doppelganger]()) and `ts` scripts (under zombie-bite-scripts), but those are designed to allow you to use (and _reuse_) each _component_ manully in order to easily debug each __step__
+
+### Requerimients
+In order to use this tool you will need this binaries available in your PATH
+
+- [Doppelganger binaries](https://github.com/paritytech/doppelganger-wrapper): doppelganger, doppelganger-parachain, workers
+- [Zombie-bite](https://github.com/pepoviola/zombie-bite)
+- Node.js (v22 or higher)
+
+This tools are installed as part of the `setup` command.
+
+### Defining the `base_path`
+
+One important concept to run all the needed steps is defining the `base_path`, this will be the directory where all the intermedia _artifacts_ will be stored, and each of the tools allow to pass the _path_ as argument.
+
+### Step 0: `bite` the live network
+
+The _first_ step consist on _biting_ a live network and ones completed create the _artifacts_ needed to _spawn_ a new instance of this network (rc/ah) and ensure block production.
+
+In order to run the _step 0_ you can run:
+
+```bash
+just zb-bite <base_path>
+
+e.g: just zb-bite ./migration-run polkadot
 ```
 
-Or, if you've already cloned and didn't use the recursive flag, you can run:
+This will run `zombie-bite`, storing the resulting _artifacts_ under `<base_path>/bite` (e.g: ./migration-run/bite) directory.
+This directory should contain these files:
+
+- `config.toml` : zombienet compatible configuration to spawn an instance of the network.
+- `<runtime>-spec.json` : chain-spec of the relaychain.
+- `asset-hub-<runtime>-spec.json` : chain-spec of AH.
+- `<runtime>-snap.tgz` : Db snapshot of the relaychain.
+- `asset-hub-paseo-snap.tgz` : Db snapshot of AH.
+
+
+_NOTE_: this step perform a _warp_ sync of both rc/ah and can take some time (20/25 mins on avg. for polkadot).
+
+
+### Step 1: `spawn` network instance (and perform migration)
+
+Ones the _step 0_ is completed, you will have all the needed artifacts for spawn a new instance of the _bited_ networks (as many times you want) with the command:
+
+```bash
+just zb-spawn <base_path>
+
+e.g: just zb-spawn ./migration-run
```

Comment:
Please double check the path. Does it have to be `migration-path-{id}` or just `{id}`? In some places we prepend `migration-path` automatically.